### ISSUE TITLE
baseURL is being ignored

### ISSUE THREAD

**Issue by tbo** [3846masa/axios-cookiejar-support#112]
The request interceptor ignores the baseURL, when building the cookie string:

https://github.com/3846masa/axios-cookiejar-support/blob/master/lib/request-interceptor-wrapper.js#L36

This solved the issue for me:
```
return getCookieString((config.baseURL ? config.baseURL : "") + config.url)
```

**Comment by 3846masa** [comment]
Thanks for issue. I'll fix it.
The fixed will release as v0.1.4.

**Comment by tbo** [comment]
Works like a charm. Thank you for the quick response!
